webapp/locales/en.json

```diff
@@ -702,7 +702,9 @@
     },
     "teaserImage": {
       "cropperConfirm": "Confirm"
-    }
+    },
+    "inappropriatePicture" : "This image may be inappropriate for some people.",
+    "inappropriatePictureText" : "When should my picture be blurred"
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-12-18T10:02:51Z" title="Wednesday, December 18th 2019, 11:02:51 am +01:00">Dec 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -702,7 +702,9 @@
     },
     "teaserImage": {
       "cropperConfirm": "Confirm"
-    }
+    },
+    "inappropriatePicture" : "This image may be inappropriate for some people.",
+    "inappropriatePictureText" : "Contribution picture is blurred"
```


and in English:
```suggestion
    "inappropriatePictureText" : "When should my picture be blurred"
```

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-12-18T11:21:06Z" title="Wednesday, December 18th 2019, 12:21:06 pm +01:00">Dec 18, 2019</time>_

---


ok